render/template.go

```diff
@@ -70,6 +75,29 @@ func (s templateRenderer) exec(name string, data Data) (template.HTML, error) {
 	return template.HTML(body), nil
 }
 
+func (s templateRenderer) assetPath(file string) string {
+
+	if len(assetMap) == 0 || os.Getenv("GO_ENV") != "production" {
+
+		log.Println("[DEBUG] reading assets manifest")
+		manifest, err := s.AssetsBox.MustString("manifest.json")
+
+		if err != nil {
+			log.Println("[INFO] didn't find manifest, using raw path to assets")
+			return fmt.Sprintf("assets/%v", file)
+		}
+
+		err = json.Unmarshal([]byte(manifest), &assetMap)
+
+		if err != nil {
+			log.Println("[Warning] seems your manifest is not correct")
+			return ""
+		}
+	}
+
+	return fmt.Sprintf("/assets/%v", assetMap[file])
```

Comment:
I expected `<%= assetPath("images/logo.svg") %>` to return `/assets/images/logo.svg`, but I just `/assets/`